smart/tests/test_map_processing.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+import pytest
+
+import astropy.units as u
+
+from sunpy.map import Map, all_coordinates_from_map, coordinate_is_on_solar_disk
+from sunpy.map.mapbase import GenericMap
+
+
+def mag_sample():
```

Comment:
```suggestion
@pytest.fixture
def hmi_nrt():
```

the rest shouldn't be fixtures - fixtures are reusable test inputs